pkg/arvo/app/file-server.hoon

```diff
@@ -276,17 +281,8 @@
         char
       (add char ^~((sub 'a' 'A')))
     ::
-    ++  get-content
-      |=  [pax=path is-file=?]
-      ^-  (unit [content path ?])
-      =/  first-try  (match-content-path pax (~(del by serving) /) is-file)
-      ?^  first-try  first-try
-      =/  root  (~(get by serving) /)
-      ?~  root  ~
-      (match-content-path pax (~(gas by *^serving) [[/ u.root] ~]) is-file)
-    ::
     ++  match-content-path
-      |=  [pax=path =^serving is-file=?]
+      |=  [pax=path =?]
```

Comment:
This line is crashing the build, it's a syntax error

Reply:
`=?` lmao